### ISSUE TITLE
Python v3.5 pip error with "black"

### ISSUE THREAD

**Issue by drewcm** [banool/recreation-gov-campsite-checker#12]
The README.md says this:
> I wrote this in Python 3.7 but I've tested it as working with 3.5 and 3.6 also.

It's worth pointing out that black (one of the project requirements) [doesn't support]() Python 3.5:

> Black can be installed by running pip install black. It requires Python 3.6.0+ to run but you can reformat Python 2 code with it, too.

When I run pip from Python v3.5.2 I get an error:
```
(venv) ~/git/recreation-gov-campsite-checker$ python --version
Python 3.5.2
(venv) ~/git/recreation-gov-campsite-checker$ pip install -r requirements.txt
...
Collecting black==18.9b0 (from -r requirements.txt (line 4))
ERROR: Could not find a version that satisfies the requirement black==18.9b0 (from -r requirements.txt (line 4)) (from versions: none)
ERROR: No matching distribution found for black==18.9b0 (from -r requirements.txt (line 4))
```

After removing black from ```requirements.txt``` I was able to run recreation-gov-campsite-checker fine.  Perhaps a note in the README.md would be sufficient to close this ticket.

It also appears that there is an unused dependency in requirements.txt: ```beautifulsoup4==4.7.1```

**Comment by banool** [comment]
I'll just update the README to say it only works with 3.6.

Thanks for pointing out the unused dependency!

**Comment by banool** [comment]
Resolved with 08b0c2abd7fdd519f904426625c0f20535d641a7 and 43942b4166df0c5126919735062748d84e620569.
